config-linux.md

```diff
@@ -455,6 +455,36 @@ The following parameters can be specified to set up the controller:
    }
 ```
 
+### <a name="configLinuxRDMA" />RDMA
+
+**`rdmaLimits`** (object, OPTIONAL) represents the cgroup subsystem `rdma`.
+For more information, see the kernel cgroups documentation about [rdma][cgroup-v1-rdma].
+
+The name of the device to limit is the entry key.
+Entry values are objects with the following properties:
+
+* **`hcaHandles`** *(uint32, OPTIONAL)* - specifies the maximum number of hca_handles in the cgroup
+* **`hcaObjects`** *(uint32, OPTIONAL)* - specifies the maximum number of hca_objects in the cgroup
+
+You MUST specify at least one of the `hcaHandles` or `hcaObjects` in a given entry, and MAY specify both.
+
+#### Example
+
+```json
+"rdmaLimits": {
```

Comment:
is there a reason why you named this `rdmaLimits` instead of just `rdma`?

Reply:
> is there a reason why you named this `rdmaLimits` instead of just `rdma`?

I like `rdma` better, although I don't care much.  Previously, @paravmellanox [cited][1] `hugepageLimits` for precedent, although the spec obviously also contains limiting properties without a `Limits` suffix.

[1]: https://github.com/paravmellanox/runtime-spec/pull/1#pullrequestreview-91323208